File: Offre-project/Offre-project/src/main/java/com/example/Offreproject/models/Offre.java
package com.example.Offreproject.models;

import lombok.*;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor

@Setter
@Getter
public class Offre {

    @Id
    @GeneratedValue(strategy =
            GenerationType.IDENTITY)
    private int Code;
    private String intitule;
    private String specialite;
    private String societe;
    private int nbPostes;
    private String pays;



}

File: Offre crud mobile/app/src/main/java/com/example/restapiwithretrofit/offre.kt
package com.example.restapiwithretrofit

data class offre (
        val code : Int? = null,
        val intitule :String? = null,
        val specialite :String? = null,
        val societe:String? = null,
        val nbPostes: Int? = null,
        val pays:String? = null,
        var isSelected:Boolean=false
        )